app/src/ui/repositories-list/group-repositories.ts

```diff
@@ -132,3 +132,61 @@ export function groupRepositories(
 
   return groups
 }
+
+/**
+ * creates the group `Recent` of repositories recently opened for use with `FilterList` component
+ *
+ * @param recentRepositories list of recent repositories' ids
+ * @param repositories full list of repositories (we use this to get data about the `recentRepositories`)
+ * @param localRepositoryStateLookup cache of local state about full list of repositories (we use this to get data about the `recentRepositories`)
+ */
+export function makeRecentRepositoriesGroup(
+  recentRepositories: ReadonlyArray<number>,
+  repositories: ReadonlyArray<Repositoryish> | null,
```

Comment:
I can't find a caller to this function that passes in `null` here, so I think this case can be omitted (and the relevant code path removed):

```suggestion
  repositories: ReadonlyArray<Repositoryish>,
```